test/lib/code_corps/helpers/policy_test.exs

```diff
@@ -0,0 +1,221 @@
+defmodule CodeCorps.Helpers.PolicyTest do
+  use CodeCorps.ModelCase
+  alias Ecto.Changeset
+  alias CodeCorps.{
+    Organization, User, Helpers.Policy,
+    ProjectUser
+  }
+
+  def create_project_user_with_role(role) do
+    user = insert(:user)
+    project = insert(:project)
+    insert(:project_user, project: project, user: user, role: role)
+    {project, user}
+  end
+
+  describe "owned_by/2" do
+    test "returns false when Organization owner_id doesn't match User user_id" do
```

Comment:
I would reword this one and the other one for organization to 

`"returns false when organization is not owned by user"`
`"returns true when organization is owned by user"`

We want the tests to talk about "what we want the app to do" not "what's implemented".